src/core/api/nodopagamenti_api/nodoPerPM/v2/_openapi.json.tpl

```diff
@@ -261,6 +261,57 @@
           "timestampOperation"
         ]
       },
+      "BancomatPayAdditionalPaymentInformations": {
+        "type": "object",
+        "description": "Additional payment data pertaining to BancomatPay payments",
+        "properties": {
+          "transactionId": {
+            "type": "string",
+            "description": "Unique identifier for the transaction, generated by the payment gateway"
+          },
+          "outcomePaymentGateway": {
+            "type": "string",
+            "enum": [
+              "OK",
+              "KO"
+            ],
+            "description": "Payment outcome"
+          },
+          "totalAmount": {
+            "type": "string",
+            "description": "sum of payment advices amount and fee, converted in string",
+            "format": "###.##",
```

Comment:
Is this format right for the total amount?

Reply:
Seeing additionalPaymentInformations example I would say yes. those fields are string converted amounts.
This structure have been cloned from paypal one's adding specific fields for bancomat pay